pkg/segment/search/conditioncheck.go

```diff
@@ -83,7 +83,8 @@ func ApplyColumnarSearchQuery(query *SearchQuery, multiColReader *segread.MultiC
 	case RegexExpression:
 		rawColVal, err := multiColReader.ReadRawRecordFromColumnFile(query.QueryInfo.ColName, blockNum, recordNum, qid)
 		if err != nil {
-			return false, err
+			log.Errorf("ApplyColumnarSearchQuery: %v", err)
```

Comment:
on a failure scenario , logging this error will cause a huge flood of the logs lines.